etc/dev-bin/updatecopyright.sh

```diff
@@ -15,12 +15,11 @@ YY=$(date +%y)
 OLD="Copyright \(C\) 2008-20\d\d NIWA & British Crown \(Met Office\) & Contributors."
 NEW="Copyright (C) 2008-20$YY NIWA & British Crown (Met Office) & Contributors."
 
-FILES=$@
-for FILE in $FILES; do
-    echo $FILE
+for FILE in "$@"; do
+    echo "$FILE"
     if [[ ! -f $FILE ]]; then
         echo "ERROR: no such file: $FILE"
         continue
     fi
-    perl -pi -e "s/$OLD/$NEW/" $FILE
+    perl -pi -e "s/$OLD/$NEW/" "$FILE"
```

Comment:
I know this isn't related to this request, but, should this perl line be replaced with a `sed -i`?

Reply:
Why should it?  (Presumably you think Perl is too heavy-weight for this?)

Reply:
Basically, yes. I'm not sure that calling perl adds any value over using sed.

Reply:
Agreed, it doesn't really.  In general, Perl is vastly more powerful, but in this trivial application it doesn't add anything.  

Note this is just a small development utility script though, used to update copyright year values once a year. So it doesn't really matter.

Reply:
At some point, we should fully eliminate our dependency on Perl.